Installer.bas

```diff
@@ -0,0 +1,52 @@
+Attribute VB_Name = "Installer"
+Option Explicit
+
+'1) Create an Excel file called Installer.xlsm in same folder than Installer.bas:
+'   *\GIT\vbaDeveloper-master\
+
+'2) Open the VB Editor (Alt+F11) right click on the active project and choose Import a file and chose:
+'    *\GIT\vbaDeveloper-master\Installer.bas
+
+'3a) Go in Tools--> References and activate:
+'   - Microsoft Scripting Runtime
+'   - Microsoft Visual Basic for Application Extensibility X.X
+
+'3b) Enable programatic access to VBA:
+'       File -> Options -> Trust Center, Trust Center Settings, -> Macros,
+'       tick the box: 'Enable programatic access to VBA'  (In excel 2010: 'Trust access to the vba project object model')
+
+'4) Run the Sub AutoInstaller in the module Installer
+
+'5) Create a new excel file and also open the file vbaDeveloper.xlam located in the folder: *\GIT\vbaDeveloper-master\
+
+'6) Make step 3a and 3b again for this file and run the sub testImport located in the module "Build".
+
+Sub AutoInstaller()
+
+'Prepare variable
+Dim CurrentWB As Workbook
+Dim NewWB As Workbook
+Dim textline As String, strPathOfBuild As String, strLocationXLAM As String
+
+'Set the variables
+Set CurrentWB = ThisWorkbook
+Set NewWB = Workbooks.Add
+
+'Import code form Build.bas  to the new workbook
+strPathOfBuild = CurrentWB.Path & "\src\vbaDeveloper.xlam\Build.bas"
+NewWB.VBProject.VBComponents.Import strPathOfBuild
+
```

Comment:
Using the import command instead of importing line by line which can cause issues for users which have Option Explicit appear at the top of every module they make. That Option Explicit would then be in conflict with the one already in the build.bas module.